src/common/options.cc

```diff
@@ -439,7 +439,7 @@ std::vector<Option> get_global_options() {
     Option("container_image", Option::TYPE_STR, Option::LEVEL_BASIC)
     .set_description("container image (used by ssh orchestrator)")
     .set_flag(Option::FLAG_STARTUP)
-    .set_default("ceph/daemon-base"),
+    .set_default("ceph/daemon-base:latest-master-devel"),
```

Comment:
How do we make sure, stable branches will use the stable images?

Reply:
This is temporary until we have a working v15 tag (octopus RC)